### PR TITLE
fix: Closes #11864 - Duplicate PropertiesModal

### LINKED ISSUE #11864: [Explore view] duplicated /api/v1/chart/id requests
#### How to reproduce the bug

1. Open explore view from chart list,
2. or Explore chart from dashboard,

### Actual results

You will see 2 /api/v1/chart/_id_ requests

### Expected results

only 1 api call for each endpoint.

#### Screenshots

Screen Shot 2020-11-30 at 2 47 56 PM

### Environment

latest master

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [x] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

### PR DESCRIPTION
**Pull request by geido**
### SUMMARY
Removes duplicate PropertiesModal import

### TEST PLAN

1. Open explore view from chart list
2. Or Explore chart from dashboard
3. Check that no duplicate /api/v1/chart/{id} is fired

### ADDITIONAL INFORMATION

- [x] Has associated issue: #11864 
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by geido** [comment]
@junlincc @rusackas

File: superset-frontend/src/explore/components/ExploreChartHeader.jsx [hunk 1]
```diff
@@ -202,7 +202,10 @@ export class ExploreChartHeader extends React.PureComponent {
             status={CHART_STATUS_MAP[chartStatus]}
           />
           <ExploreActionButtons
-            actions={this.props.actions}
+            actions={{
+              ...this.props.actions,
+              openPropertiesModal: this.openProperiesModal,
```
Comment (nikolagigic) [inline superset-frontend/src/explore/components/ExploreChartHeader.jsx:207]: typo in `this.openProperiesModal` . Should be `this.openPropertiesModal`

Comment (geido) [reply superset-frontend/src/explore/components/ExploreChartHeader.jsx:207]: Yeah, that was there already, but let me fix it. I also hate typos :)

**Comment by villebro** [comment]
I can confirm that this PR fixes it (the chart id here is 989):

# Before
image

# After
image

Reviewing code now.

**Review by villebro — APPROVED** [review]
LGTM - thanks for the fix (both bug and typos)!

**Pull request merged**